### DISCUSSION TITLE
genome resource of other species

### DISCUSSION THREAD

**Discussion by pengxin2019** [GreenleafLab/ArchR#940] [Questions / Documentation]
Dear ArchR development team,
I am handling scATAC data of other species. (neither human nor mouse).
From the document of addArchRGenome function, I realize that it is possible to create this  global environment variable representing the genome: https://www.archrproject.com/reference/addArchRGenome.html

Taking `createArrowFiles` as example, https://www.archrproject.com/reference/createArrowFiles.html
It seems like both the input of the `getGeneAnnotation` and `getGenomeAnnotation` is an ArchRProject object. But I did not find the function to create this ArchRProject object using a non-ArrowFiles. I find `ArchRProject` but is requires a `ArrowFiles` as input. I can query the gene annotation from other packages like `AnnotationHub` but I do not know how to make it to be a ArchRProject object.

Also, in terms  of `createGenomeAnnotation` at https://www.archrproject.com/reference/createGenomeAnnotation.html
If I remember correctly, the source of the BSgenome is UCSC not ensembl which is what we preferred in terms of the species we are working on. It seems like only a BSgenome object  could be accepted by createGenomeAnnotation. Providing the http to the gtf file of the genome or annotation would not work, right? 
Is creating a BSgenome by myself the only solution that might work?

Thanks
Best
PENNY

**Comment by rcorces** [comment]
Please see the manual https://www.archrproject.com/bookdown/getting-set-up.html section 1.5.1
